tests/cupy_tests/manipulation_tests/test_kind.py

```diff
@@ -10,6 +10,20 @@
 @testing.gpu
 class TestKind(unittest.TestCase):
 
+    @testing.for_all_dtypes()
+    def test_asarray_chkfinite(self, dtype):
+        a = [0, 4, 0, 5]
+        a_gpu = cupy.asarray_chkfinite(a, dtype=dtype)
+        a_cpu = numpy.asarray_chkfinite(a, dtype=dtype)
+        assert a_gpu.dtype == a_cpu.dtype
+
+    @testing.for_all_dtypes()
+    def test_asarray_chkfinite_non_finite_vals(self, dtype):
+        a = [-numpy.inf, 0., numpy.inf]
+        a_gpu = cupy.asarray_chkfinite(a, dtype=dtype)
+        a_cpu = numpy.asarray_chkfinite(a, dtype=dtype)
```

Comment:
I think this too will raise an error and you won't get anything in `a_cpu`. As nothing would be returned.
I'm not sure you need to test this or even write this.

Reply:
Yes, `a_cpu` too will raise value error.